main/modes/games/bigbug/mode_bigbug.c

```diff
@@ -21,11 +21,15 @@
 #include "esp_heap_caps.h"
 #include "hdw-tft.h"
 #include <math.h>
+#include "mainMenu.h"
 
 //==============================================================================
 // Defines
 //==============================================================================
 
+/** Switch to either start at the menu or on the dump's surface */
+// #define SKIP_INTRO
```

Comment:
Here's a single switch for skipping the intro. Using the preprocessor means no warning for an unused function!